setup.cfg

```diff
@@ -55,6 +55,8 @@ test =
 typing =
     mypy==0.790
     typing-extensions>=3.7.4.3
+virtualenv =
```

Comment:
This extra allows users to opt-in into a faster isolated env creator.

Reply:
Why is `venv` slower?

Reply:
see https://virtualenv.pypa.io/en/latest/user_guide.html#seeders, but boils down to virtualenv using a cached seed technique, venv does not